code/src/components/TaskList.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import tasks from 'reducers/tasks'
+import styled from 'styled-components'
+import trashBin from 'Assets/trashBin.png'
+import NewTask from 'components/NewTask'
+
+const Tasks = () => {
+  const allTasks = useSelector((store) => store.tasks.todos);
+  const dispatch = useDispatch();
+
+  const onIsTaskDone = (id) => {
+    dispatch(tasks.actions.markTaskAsDone(id))
+  }
+  const onRemoveTaskClick = (task) => {
+    dispatch(tasks.actions.removeTask(task))
+  }
+  return (
+    <Outerwrapper>
+      <InnerWrapper>
+        {allTasks.map((singleTask, index) => {
+          return (
+            <Content key={singleTask.id}>
+              <input
+                type="checkbox"
+                id="checkbox"
+                checked={singleTask.isDone}
+                onChange={() => onIsTaskDone(singleTask.id)} />
+              <label>
+                <TasklistText>{singleTask.title}</TasklistText>
+              </label>
+              <RemoveTaskButton type="button" onClick={() => onRemoveTaskClick(index)}>
+                <Icon
+                  className="icon1"
+                  src={trashBin}
+                  alt="IconTrash" />
+              </RemoveTaskButton>
+            </Content>
+          )
+        })}
+        <NewTask />
+      </InnerWrapper>
+    </Outerwrapper>
+  )
+}
+
+export default Tasks
+
+const Outerwrapper = styled.section`
+  background-color: #5bc0be;
+  /* width: 100%;
+  height: 80vh; */
+  display: flex;
+  margin-top: 30px;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+`
+
+const InnerWrapper = styled.section`
+  width: 80%;
+  background-color: #409A98;
+  border-radius: 20px;
+  box-shadow: 5px 5px 5px #2f4858;
+  @media (min-width: 668px) and (max-width: 1024px) {
+    width: 40%;
+  }
+  @media (min-width: 1025px) {
+    width: 30%;
+  }
+  `
```

Comment:
These could be a bit bigger, like 50% (at least on my screen the list is a bit small, but that might also be just an individual preference)